recipes-security/optee-imx/optee-os_3.17.0.imx.bb

```diff
@@ -60,20 +61,29 @@ EXTRA_OEMAKE = " \
     -C ${S} O=${B} \
 "
 
-LDFLAGS = ""
+LDFLAGS[unexport] = "1"
 CFLAGS += "--sysroot=${STAGING_DIR_HOST}"
 CXXFLAGS += "--sysroot=${STAGING_DIR_HOST}"
 
+do_configure[noexec] = "1"
+
+do_compile:arm () {
+    oe_runmake all uTee
+}
+
+do_compile:aarch64 () {
+    oe_runmake all
+}
+do_compile[cleandirs] = "${B}"
```

Comment:
Isn't this default?

Reply:
Seems not?

```
sources$ grep -r do_compile\\[clean poky/
poky/meta/recipes-devtools/go/go-cross.inc:do_compile[cleandirs] += "${GOTMPDIR} ${B}/bin ${B}/pkg"
poky/meta/recipes-devtools/go/go-cross-canadian.inc:do_compile[cleandirs] += "${GOTMPDIR} ${B}/bin ${B}/pkg"
poky/meta/recipes-devtools/go/go-crosssdk.inc:do_compile[cleandirs] += "${GOTMPDIR} ${B}/bin ${B}/pkg"
poky/meta/recipes-devtools/go/go-native_1.19.bb:do_compile[cleandirs] += "${GOTMPDIR} ${B}/bin"
poky/meta/recipes-devtools/go/go-target.inc:do_compile[cleandirs] += "${GOTMPDIR} ${B}/bin ${B}/pkg"
poky/meta/recipes-devtools/go/go-runtime.inc:do_compile[cleandirs] += "${GOTMPDIR} ${B}/bin ${B}/pkg"
poky/meta/classes-recipe/python_pep517.bbclass:do_compile[cleandirs] += "${PEP517_WHEEL_PATH}"
poky/meta/classes-recipe/setuptools3.bbclass:do_compile[cleandirs] += "${PEP517_WHEEL_PATH}"
poky/meta/classes-recipe/go.bbclass:do_compile[cleandirs] = "${B}/bin ${B}/pkg"
```

We got this from `meta-arm`:

https://github.com/jonmason/meta-arm/commit/12604dc9d3cb2348c002a5da9858ec9f59dc503a